cypress/pageObject/Users/UserAvatar.ts

```diff
@@ -0,0 +1,59 @@
+export class UserAvatar {
+  username: string;
+  constructor(username: string) {
+    this.username = username;
+  }
+
+  navigateToProfile() {
+    cy.visit(`/users/${this.username}`);
+    return this;
+  }
+
+  interceptUploadAvatarRequest() {
+    cy.intercept("POST", `/api/v1/users/${this.username}/profile_picture/`).as(
+      "uploadAvatar",
+    );
+    return this;
+  }
+
+  clickChangeAvatarButton() {
+    cy.verifyAndClickElement('button[id="change-avatar"]', "Change Avatar");
```

Comment:
```suggestion
    cy.verifyAndClickElement('button[id="change-avatar"]', "Change Avatar");
```

change the entire id and use the data-cy for the element